root.json

```
@@ -1,4 +1,5 @@
 {
+    "F2Auth": "F2Auth.json",
```

```suggestion
    "2FAuth": "2FAuth.json",
```
@Hooverdan96
This is what I'm hoping we need here :).